browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -356,6 +356,10 @@ var NotebookbarAccessibility = function() {
 					  this.focusToMap();
 					}
 				  }.bind(this));
+
+				if (element.classList.contains('selected')) {
+					this.setupAcceleratorsForCurrentTab(element.id);
+				}
```

Comment:
Thanks for the improvements :)
The reason we removed this from here was: "to not assign accelerator keys before a tab is selected."
If we add this again, there may be conflicts with the tab header accelerator keys.

I think we can modify below part:
    "else if (key === 'ARROWDOWN') {"

We can make it work regardless of the active tab pointers content list. You can check the current tab and find its content and focus on the first element. Without using "this.activeTabPointers.contentList" or any other variable from within the class. Can you check if it is possible?

I will be continuing with the second commit.